academy/4-my-own-chain/wager-denom.md

```diff
@@ -101,27 +101,31 @@ You have prepared the ground by placing this token denomination into the relevan
+A relayer is a process that transfers IBC packets between two blockchains. It just so happens that here this process is **running in your browser** using the account you configured in your browser. The account is the same one you would use to play a game of checkers. Name it `alice123@checkers`.
 
-1. On the checkers end, the relayer is already configured to connect to your running checkers blockchain, and to use the tokens of whichever account you have configured in your browser, here `alice123@checkers`. In this sense, it gets the same privileges to access your tokens that you have granted to the checkers browser application.
-2. You now need to configure it to connect to the other blockchain, the one that hosts the foreign tokens you want to transfer. It can be the Cosmos Hub, or a [testnet](https://tutorials.cosmos.network/connecting-to-testnet/testnet-tutorial.html) that you or someone else runs.
-3. You also need to fund the relayer's account on the remote chain so that it can operate at all. This account is generated from the same private key as `alice123@checkers`, so let's call it `alice465@remote`. The relayer shows you in the browser which account this is.
+1. On the checkers end, the relayer is already configured to connect to your running checkers blockchain and to use the tokens of whichever account you have configured in your browser, here `alice123@checkers`. It gets the same privileges to access your tokens that you have granted to the checkers' browser application.
+2. You need to configure it to connect to the other blockchain hosting the foreign tokens you want to transfer. It can be the Cosmos Hub or a [testnet](https://tutorials.cosmos.network/connecting-to-testnet/testnet-tutorial.html) that you or someone else runs.
+3. You also need to fund the relayer's account on the remote chain so that it can operate. The account is generated from the same private key as `alice123@checkers`, so call it `alice465@remote`. The relayer shows you in the browser which account this is.
 
 Your test comes in a few steps:
 
-1. Configure the relayer. This is a matter of entering the necessary parameters, clicking a button and waiting for the setup to be done. In effect, the relayer opened a transfer channel, likely numbered `0`, on the checkers chain, and opened another transfer channel on the remote chain, and linked the two.
-2. Send the desired foreign tokens to your `alice465@remote`, using any regular method of sending tokens, be it from a faucet or from another account of yours.
+1. Configure the relayer. This is a matter of entering the necessary parameters, clicking a button, and waiting for the setup to be done. The relayer opened a transfer channel on the checkers chain likely numbered `0`, opened another transfer channel on the remote chain, and linked both chains.
```

Comment:
By moving "likely numbered 0" around, you increased confusion and the likelihood that the reader will pause to think "wait what is numbered 0? is it the chain, whose word is 1 word before, or is it the channel, whose word is 5 words before?".
Before thinking "it must be the channel, ok, back to reading. What was it about again? Ok better restart the paragraph from the start". 

Also, it is the channels that are linked, not the chains.

Reply:
Will change accordingly in my next commit.